build/Helix/CreateHelixTestFailureDumps.ps1

```diff
@@ -6,6 +6,24 @@ Param(
     [string]$OutputFilePath = "dumplinks.html"
```

Comment:
Also rename the output file to something more descriptive. LinksToHelixTestFiles.html or something like that.